poms/bom-test/pom.xml

```diff
@@ -48,7 +48,7 @@
             <dependency>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-bom-test</artifactId>
-                <version>${quarkus.version}</version>
+                <version>${quarkus-community.version}</version>
```

Comment:
there's no quarkus-bom-test in the Red Hat build ? Or just for this version?

Reply:
No they do not productize test artifacts intentionally.

Reply:
Ok good to know